### PR TITLE
Identity api req set

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[acc8e8e](https://github.com/OfficeDev/office-js-docs-pr/commits/acc8e8eb182e8c087dff2a9be6ca119f37bf0904)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201911120046306144-1432%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3f%2F1dVbDLWZysE2CKDRu7DAXnwjDpT5f00yqOgNZU5Y%3D&st=2019-11-12T00%3A42%3A43Z&se=2019-12-13T00%3A47%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/overview-authn-authz.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ You can get authorization to Microsoft Graph data for your add-in by obtaining a
...
+To use SSO to authenticate the user, your code in a task pane or function file calls the [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-) method. If the user is not signed into Office, Office will open a dialog and navigate it to the Azure Active Directory login page. After the user is signed in, or if the user is already signed in, the method returns an access token. The token is a bootstrap token in the **On Behalf Of** flow. (See [Access to Microsoft Graph with SSO](#access-to-microsoft-graph-with-sso).) However, it can be used as an ID token as well, because it contains several claims that are unique to the current user, including `preferred_username`, `name`, `sub`, and `oid`. For guidance on which property to use as the ultimate user ID, see [Microsoft identity platform access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens#payload-claims). For an example of a one of these tokens, see the [Example access token](sso-in-office-add-ins.md#example-access-token).
```
Comment (kbrandl) [inline docs/develop/overview-authn-authz.md]: The bookmark portion of the link target `/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-)` needs to be all lowercase in order for the bookmark to work properly.

File: docs/develop/overview-authn-authz.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ These samples, however, do not use the token as an ID token. They use it to get
...
+To use SSO to get access to Microsoft Graph, your add-in in a task pane or function file calls the [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-) method. If the user is not signed into Office, Office will open a dialog and navigate it to the Azure Active Directory login page. After the user is signed in, or if the user is already signed in, the method returns an access token. The token is a bootstrap token in the **On Behalf Of** flow. Specifically, it has a `scope` claim with the value `access_as_user`. For guidance about the claims in the token, see [Microsoft identity platform access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens#payload-claims). For an example of a one of these tokens, see the [Example access token](sso-in-office-add-ins.md#example-access-token).
```
Comment (kbrandl) [inline docs/develop/overview-authn-authz.md]: The bookmark portion of the link target `/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-)` needs to be all lowercase in order for the bookmark to work properly.

i.e., it should be:   `/javascript/api/office-runtime/officeruntime.auth#getaccesstoken-options-)`

File: docs/develop/sso-in-office-add-ins.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ The following diagram shows how the SSO process works.
 
 ![A diagram that shows the SSO process](../images/sso-overview-diagram.png)
 
-1. In the add-in, JavaScript calls a new Office.js API [getAccessToken](/javascript/api/office/officeruntime.auth#getAccessToken-options--callback-). This tells the Office host application to obtain an access token to the add-in. See [Example access token](#example-access-token).
+1. In the add-in, JavaScript calls a new Office.js API [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-). This tells the Office host application to obtain an access token to the add-in. See [Example access token](#example-access-token).
```
Comment (kbrandl) [inline docs/develop/sso-in-office-add-ins.md]: bookmark portion of the link target needs to be all lowercase -> `#getaccesstoken-options-`

File: docs/develop/sso-in-office-add-ins.md [hunk 4]
```diff
@@ -91,7 +91,7 @@ The following is an example of the markup:
 
 Add JavaScript to the add-in to:
 
-* Call [getAccessToken](/javascript/api/office/officeruntime.auth#getAccessToken-options--callback-).
+* Call [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-).
```
Comment (kbrandl) [inline docs/develop/sso-in-office-add-ins.md]: bookmark portion of the link target needs to be all lowercase -> `#getaccesstoken-options-`

File: docs/develop/sso-in-office-add-ins.md [hunk 5]
```diff
@@ -222,10 +222,10 @@ There are some small, but important differences in using SSO in an Outlook add-i
 
 ### getAccessToken
 
-The OfficeRuntime [Auth](/javascript/api/office/officeruntime.auth) namespace, `OfficeRuntime.auth`, provides a method, `getAccessToken` that enables the Office host to obtain an access token to the add-in's web application. Indirectly, this also enables the add-in to access the signed-in user's Microsoft Graph data without requiring the user to sign in a second time.
+The OfficeRuntime [Auth](/javascript/api/office-runtime/officeruntime.auth) namespace, `OfficeRuntime.auth`, provides a method, `getAccessToken` that enables the Office host to obtain an access token to the add-in's web application. Indirectly, this also enables the add-in to access the signed-in user's Microsoft Graph data without requiring the user to sign in a second time.
```
Comment (kbrandl) [inline docs/develop/sso-in-office-add-ins.md]: should `OfficeRuntime.auth` instead be `OfficeRuntime.Auth` (capital A)?

image

Comment (kbrandl) [inline docs/develop/sso-in-office-add-ins.md]: should probably a comma after `getAccessToken`

**Comment by o365devx** [comment]
Docs Build status updates of commit _[cc282c3](https://github.com/OfficeDev/office-js-docs-pr/commits/cc282c3f6e680894c352c6793cb2ddf6f9027bae)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201911122329036954-1432%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=T9KcDQrPyvqalkJ2BqenFts4KWLOaBlIlGHY4j2b9qA%3D&st=2019-11-12T23%3A25%3A15Z&se=2019-12-13T23%3A30%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/overview-authn-authz.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/develop/overview-authn-authz.md]: fixed

**Comment by o365devx** [comment]
Docs Build status updates of commit _[666191d](https://github.com/OfficeDev/office-js-docs-pr/commits/666191ddff7a90830a1ff0563cec712f2478e234)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/identity-api-req-set/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201911122335384059-1432%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=j234NP2A%2F5SU1o3FSi9%2FEnfb2VLyF8uIiPdF4heyd8I%3D&st=2019-11-12T23%3A31%3A49Z&se=2019-12-13T23%3A36%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**